### ISSUE TITLE
Edcast Standalone fails to send metadata

### ISSUE THREAD

**Issue by GoogleCodeExporter** [andrew867/edcast-reborn#65]
```
Hey guys!
I've got a stream set up, however the artist and title info isn't being sent to 
the SHOUTcast server, apparently.

I've got Edcast Standalone 3.37.2011.1214 set to broadcast the title of a 
Foobar 2000 window playing music. I know I set this up correctly, as the artist 
and title information is being updated in the actual metadata window in Edcast. 
But it's not going to the server - in Winamp, it displays just the server name 
(that works). In playing with it I was able to get artist and title info for 
the current song to successfully show once, and only once in Winamp.
Also, I know it's not the server, as the Edcast Foobar 2000 plugin I've got 
works perfectly in regards to metadata.

Is this a known bug?
```

Original issue reported on code.google.com by `Kevan...@gmail.com` on 30 Oct 2012 at 2:02

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem ! :/
```

Original comment by `kustic.m...@gmail.com` on 25 Jan 2013 at 11:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `jaroma...@gmail.com` on 9 Feb 2013 at 2:35
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
Original comment by `jaroma...@gmail.com` on 9 Feb 2013 at 2:39
- Changed state: **WontFix**
